source/aws-bootstrap-kit/API.md

```diff
@@ -8,6 +8,7 @@ Name|Description
 [AwsOrganizationsStack](#aws-bootstrap-kit-awsorganizationsstack)|A Stack creating the Software Development Life Cycle (SDLC) Organization.
 [CrossAccountDNSDelegator](#aws-bootstrap-kit-crossaccountdnsdelegator)|TODO: propose this to fix https://github.com/aws/aws-cdk/issues/8776 High-level construct that creates: 1. A public hosted zone in the current account 2. A record name in the hosted zone id of target account.
 [RootDns](#aws-bootstrap-kit-rootdns)|A class creating the main hosted zone and a role assumable by stages account to be able to set sub domain delegation.
+[SecureRootUser](#aws-bootstrap-kit-securerootuser)|*No description*
```

Comment:
Is this intentional?

Reply:
Yes, I think at some point we will have to expose everything. Here it's specifically needed to be able to write integTests